adafruit_ble/advertising/__init__.py

```diff
@@ -206,8 +206,8 @@ class Advertisement:
     """
 
     match_prefixes = ()
-    """For Advertisement, `Advertisement.matches` will always return ``True``.
-    Subclasses may override this value."""
+    """For Advertisement, :py:attr:`~adafruit_ble.advertising.Advertisement.match_prefixes`
```

Comment:
Try leaving off the `:py:attr:`. I think it will still work and be more readable.
```suggestion
    """For Advertisement, `~adafruit_ble.advertising.Advertisement.match_prefixes`
```